treeAnalysis/treeProcessing.h

```diff
@@ -96,9 +152,9 @@ float* _track_Proj_true_z             = new float[_maxNProjections];
 float* _track_Proj_true_t             = new float[_maxNProjections];
 // MC particles
 int _nMCPart;
-float* _mcpart_ID                = new float[_maxNMCPart];
-float* _mcpart_ID_parent         = new float[_maxNMCPart];
-float* _mcpart_PDG               = new float[_maxNMCPart];
+int* _mcpart_ID                = new int[_maxNMCPart];
+int* _mcpart_ID_parent         = new int[_maxNMCPart];
+int* _mcpart_PDG               = new int[_maxNMCPart];
```

Comment:
In the production I have, this branch is floats so reading it as ints produces bad data.  Has something changed upstream to match this?

Reply:
Yeah, it's been changed upstream to ints (as they really should be), but that hasn't yet propagated to the productions. Since we're working on both development and the productions, I would suggest just making the changes in your local branch. I'm usually carrying around a few minor changes like this (not ideal, but when types changes, there's not a lot that can be done easily here, I think)

Reply:
Makes sense, I mostly wanted to verify it was intentional